tests/matrix_multiply_tests/matrix_multiply.futil.data

```diff
@@ -0,0 +1,26 @@
+{
+  "arr_1": {
+    "data": [ [5, 3, 2, 6], [4, 9, 2, 1], [3, 7, 2, 6], [6, 1, 4, 2] ],
```

Comment:
You may want to consider formatting this file so it looks like the `.expect` file for the sake of readability and easy comparison! Basically you want to format this as a JSON file, so if you have a formatter you just need to tell your formatter that the file is (despite its extensions) actually a JSON file.